source/renderer/app/components/wallet/settings/DelegateWalletButton.scss

```diff
@@ -0,0 +1,8 @@
+.root {
+  font-weight: 500 !important;
+  height: 36px !important;
+  line-height: 36px !important;
+  margin-left: 20px;
+  min-width: 160px !important;
+  width: 160px !important;
+}
```

Comment:
@yakovkaravelov can you wrap component and use global to override this instead of `!important`
e.g.

```
.componentWrapper {
  :global {
      .Button_root {...}
  }
}
```

Reply:
@tomothespian Good catch. I didn't make this one new in this PR. Just copied from other existing style, and didn't try to beautify it. Will fix now.